bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -0,0 +1,208 @@
+#!/bin/env python
+""" Apply a naive mass binning, assuming that each bin is fully independent, which 
+is a conservative estimate. This clusters to find the most significant foreground, but
+leaves the background triggers alone. """
+
+import h5py, numpy, argparse, logging, pycbc, pycbc.events, lal
+import pycbc.version
+from scipy.special import comb
+import matplotlib as mpl
+mpl.use('agg')
+import matplotlib.pyplot as plt
+
+def com(f, files, group):
+    """ Combine the same column from multiple files and save to a third"""
+    f[group] = numpy.concatenate([fi[group][:] if group in fi else \
+                                   numpy.array([], dtype=numpy.uint32) for fi in files])
+
+def all_keys_set(files):
```

Comment:
what this actually does is return unique names of all h5py datasets .. would prefer to call it e.g. 'name_all_datasets'